MQuince/MQuince.Integration.HospitalApp.Tests/OperationsWithDatabaseTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Moq;
+using MQuince.Integration.Repository.Contracts;
+using MQuince.Integration.Services.Constracts.DTO;
+using MQuince.Integration.Services.Implementation;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MQuince.HospitalAppTests
+{
+    public class OperationsWithDatabaseTest
+    {
+        [Fact]
+        public void Create_entity_in_mock_database()
+        {
+            var stubRepository = new Mock<IPharmacyRepository>();
+
+            PharmacyDTO pharmacy = new PharmacyDTO()
+            {
+                Name = "some_name"
+            };
+
+            PharmacyService service = new PharmacyService(stubRepository.Object);
+
+            Assert.NotNull(service.Create(pharmacy));
```

Comment:
Assert doesn't check if repository state is changed, just that returned value is not null